AUTHORS.md

```diff
@@ -0,0 +1,22 @@
+`romcal` was orginally developed by **Julian Pereira** from Malaysia in July 2014.
+
+Since then, many significant contributions have been made towards the progress of this library.
+
+This list recognizes contributors to `romcal` past and present.
```

Comment:
As we have agreed some time ago, romcal is the name of this library and we don’t need to style it as `code`.

Reply:
Done!